assets/js/modules/analytics/components/module/ModulePopularPagesWidgetGA4.js

```diff
@@ -188,7 +188,7 @@ function ModulePopularPagesWidgetGA4( props ) {
 			hideOnMobile: true,
 			field: 'metricValues.2.value',
 			Component: ( { fieldValue } ) => (
-				<span>{ numFmt( fieldValue, { style: 'decimal' } ) }</span>
+				<span>{ numFmt( Number( fieldValue ) / 100, '%' ) }</span>
```

Comment:
Should we maybe use `parseFloat` instead?

Also, `fieldValue` for `engagementRate` will be a number between 0 and 1 (inclusive) so we shouldn't need to divide by 100.

Reply:
It doesn't look like we need `parseFloat()` here, just passing `fieldValue` in to `numFmt()` without dividing it, while specifying the `%` format seems fine. Note that we pass `fieldValue` directly to `numFmt()` in the other cases too, and these are all copied from the UA version of the widget.